src/ceph.in

```diff
@@ -552,8 +552,8 @@ def ping_monitor(cluster_handle, name, timeout):
         return 1
 
     mon_id = name[len('mon.'):]
+    run_in_thread(cluster_handle.connect, timeout=timeout)
```

Comment:
Perhaps the right fix is in build_initial_monmap, then, to actually use mon_initial_members and/or the DNS lookup it does